vhost_user_fs/src/filesystem.rs

```diff
@@ -1043,6 +1045,31 @@ pub trait FileSystem {
         Err(io::Error::from_raw_os_error(libc::ENOSYS))
     }
 
+    /// Setup a mapping so that guest can access files in DAX style.
+    #[allow(clippy::too_many_arguments)]
+    fn setupmapping<T: FsCacheReqHandler>(
```

Comment:
s/setupmapping/setup_mapping/

Reply:
Or could it just be 'map' ?

Reply:
I'm not against renaming it, but what I did was to follow how CLH has converted virtiofsd's operation names, e.g.
-  do_releasedir -> fn releasedir(), 
- do_fsyncdir -> fn fsyncdir()

Do you think we need to make an exception here?

Reply:
If it's consistent with the rest of the file then I guess that's ok.